### DISCUSSION TITLE
Adding placeholder files in repo and expanding documentation

### DISCUSSION THREAD

**Discussion by SpasovskiFilip** [alexplesoiu/dns-updater#4] [Ideas]
So my idea is this:
If the the `main.py` uses the `.env` file for the variables, we should put a placeholder `.env` file in the repository, so whoever uses it can see an example of how it's done and just replace the placeholder values with their own keys.

Also would be good to add the option to the docker run to bind that file in the docker container with the `volume` command.
While talking about docker, I think we can also add a docker compose example in the documentation / readme file.

Please feel free to comment and expand on this Idea

**Comment by alexplesoiu** [comment]
Definitely we should use environment variables and extend the support via docker-compose, so everyone can pull a docker image and configure it via env vars or via local files.

Also, until then, can you modify just the readme file to also include the tutorial for cloud flare api keys? 

I have a few ideas for future development, so I will create a new topic for the next release.

**Reply by SpasovskiFilip** [reply]
Sure thing. Done.
